rewrite-core/src/main/java/org/openrewrite/TreeVisitor.java

```diff
@@ -76,7 +76,12 @@ public final Cursor getCursor() {
     }
 
     @Nullable
-    public T visitEach(T tree, P p) {
+    public T preVisit(T tree, P p) {
+        return defaultValue(tree, p);
+    }
+
+    @Nullable
+    public T postVisit(T tree, P p) {
         return defaultValue(tree, p);
```

Comment:
just highlighting-- acceptable for `postVisit` to call `defaultValue` same as `preVisit`, would assume?

Reply:
Yes. That is the correct behavior.